app/views/footer.html

```diff
@@ -26,10 +26,10 @@
         <p class="block-title">Contact</p>
 
         <div class="clearfix well">
-          <img src="/images/julie.jpg" class="img-responsive pull-right" alt="Julie Meiresonne">
-          <p>Julie Meiresonne</p>
+          <img src="/images/wim.jpg" class="img-responsive pull-right" alt="Wim Dhaeyer">
```

Comment:
Nit: I'm not sure this `alt` makes sense since Wim's name is directly underneath, so screen readers would read `Wim Dhaeyer` twice. This feels like a decorative image? https://www.anysurfer.be/nl/documentatie/artikels/detail/tekstalternatieven#deco

> Als de afbeelding informatie bevat die dubbelop is met de tekst, kunnen we ze ook als decoratief beschouwen.
> [...] Ze moet worden genegeerd door hulpmiddelen. Dat doe je meestal door een lege alternatieve tekst te geven.

Reply:
Referencing anysurfer because that's _the_ authority for accessible websites in Belgium and we often ask their input on websites/services that we build.

Reply:
I removed the alt.

Reply:
Ah, it should actually be `alt=""` for decorative images 😉 https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Sorry I thought that was clear from `Dat doe je meestal door een lege alternatieve tekst te geven.` but I can see the confusion now

Reply:
My bad 🤦‍♂️

Reply:
All good now! 👍